enterprise_catalog/apps/catalog/content_metadata_utils.py

```diff
@@ -71,9 +71,11 @@ def is_course_run_active(course_run):
     Returns:
         bool: True if course run is "active"
     """
+    is_enrollable = course_run.get('is_enrollable', False)
+    if course_run.get("is_marketable_external") and is_enrollable:
+        return True
     course_run_status = course_run.get('status') or ''
     is_published = course_run_status.lower() == 'published'
-    is_enrollable = course_run.get('is_enrollable', False)
     is_marketable = course_run.get('is_marketable', False)
 
     return is_published and is_enrollable and is_marketable
```

Comment:
Nit: instead of returning early, I think I'd rather see this pattern which should behave the same:
```suggestion
    is_marketable_internal = is_published and is_marketable
    return is_enrollable and (is_marketable_external or is_marketable_internal)
```